packages/merkleized-metadata/src/transform/metadata.ts

```diff
@@ -191,7 +191,7 @@ export function generateExtrinsicMetadata(metadata: Metadata, accessibleTypes: M
   const extrinsic = metadata.latest.extrinsic;
 
   return {
-    version: extrinsic.version,
+    version: extrinsic.version.at(0) || 0,
```

Comment:
Not sure if we'll have an update for RFC-0078 for this change since we'll have multiple versions here, for now I'm good starting with this. 

```suggestion
    import { ensurePresence } from '@dedot/utils';
    version: ensurePresence(extrinsic.version.at(0), 'Extrinsic version not found from metadata'),
```